portal/views.py

```diff
@@ -62,20 +63,20 @@ def get_public_apps(request, app_id=0, collection=None, order_by="updated_on", o
 
     published_apps = []
 
+    app_orms = (app_model for app_model in APP_REGISTRY.iter_orm_models() if issubclass(app_model, SocialMixin))
     if collection:
         # TODO: TIDY THIS UP!
-
-        for subclass in SocialMixin.__subclasses__():
-            print(subclass, flush=True)
-            published_apps_qs = subclass.objects.filter(
+        for app_orm in app_orms:
+            print(app_orm, flush=True)
```

Comment:
Why not use logging instead of print?

Reply:
It was given like that, but I can change it:)